sakuya/mewo.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
```

Comment:
style nit for the entire file (why doesn't GitHub allow file-level comments...): I believe this project (mostly) uses single quotes, so change all double-quotes to single-quotes in this file

Reply:
Quotes follow the same-ish pattern from other files in the project; where fixed-strings are generally single-quotes and formated strings/client responses are double